cromulent/model.py

```diff
@@ -434,14 +438,22 @@ def toFile(self, what, compact=True, filename="", done=None):
 		fh.close()
 		return out
 
-	def production_mode(self):
-		self.cache_hierarchy()
-		self.validate_profile = False
-		self.validate_properties = False
-		self.validate_range = False
-		self.validate_multiplicity = False
-		return True
-
+	def production_mode(self, state="on"):
```

Comment:
Are t there ever going to be more than two states? Could/should this be a `bool` flag argument?

Reply:
My tendency is to use named flags in case there are other options. Perhaps we need to set the flags on the factory, but not cache the hierarchy because once cached, you can't reverse the operation.

But I don't have a specific requirement here.